enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -664,9 +664,10 @@ Users can have multiple shared Gateways, similar to the concept of [sharding](ht
 
```

Comment:
I suggested removing the paragraph in https://github.com/openshift/enhancements/pull/1741/files#r2006517621:

This cross-namespace topology can only be implemented using the Gateway-to-xRoute hierarchy, as Istio does not support
cross-namespace Gateway merging. Therefore, a limitation of this topology is that xRoutes lack an API for
specifying TLS certificates, requiring all application certificates to be managed within a cluster-wide Gateway.